ketanjamnal/Task2.html

```diff
@@ -0,0 +1,37 @@
+<html>
+
+<head>
+    <title>Retro City</title>
+    <style>
+        .container {
+            position: relative;
+        }
+
+        
+        .text-block {
+            position: absolute;
+            bottom: 20px;
+            right: 20px;
+            background-color: rgba(248, 135, 6, 0.89);
+            color: rgb(5, 5, 5);
+            padding-left: 20px;
+            padding-right: 20px;
+            height: 70px;
+            width: 200px;
+            box-shadow: 10px 10px lightblue;
+            
+
+        }
+    </style>
+</head>
+
+<body>
+    <div class="container">
+        <img width="100%" src="task2.jpg" alt="Retro City">
```

Comment:
Remove the file and either use `../assets/images/task2.jpg` or `https://github.com/CC-BHU/web-development/raw/main/assets/images/task2.jpg`